src/Altinn.Apps/AppTemplates/AspNet/Altinn.App.PlatformServices/Implementation/AppBase.cs

```diff
@@ -51,17 +50,12 @@ public abstract class AppBase : IAltinnApp
         public abstract Task RunDataCreation(object instance);
         
         /// <inheritdoc />
-        public async Task<string> OnInstantiateGetStartEvent(Instance instance)
+        public Task<string> OnInstantiateGetStartEvent()
         {
-            _logger.LogInformation($"OnInstantiate for {instance.Id}");
+            _logger.LogInformation($"OnInstantiate for ");
```

Comment:
Do we still need this log message? can we make it more descriptive if we do now that we don't have the instance Id?